### ISSUE TITLE
An internal error occurred during: "Launching XXXX".

### ISSUE THREAD

**Issue by thugcee** [bndtools/bndtools#1836]
the only thing I get in log is this:
```
!ENTRY org.eclipse.core.jobs 4 2 2018-02-27 14:24:57.860
!MESSAGE An internal error occurred during: "Launching edu.zut.aci.nsg.restapi (1)".
!STACK 0
java.lang.NullPointerException
        at bndtools.launch.OSGiRunLaunchDelegate.getLauncherStatus(OSGiRunLaunchDelegate.java:98)
        at bndtools.launch.AbstractOSGiLaunchDelegate.finalLaunchCheck(AbstractOSGiLaunchDelegate.java:100)
        at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:868)
        at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
        at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
        at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
        at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
```

I don't know where to look for a source of the problem. The project has been created by adding _Bndtools Project Nature_ to standard Java Gradle project (OSGI bundle was built using Gradle.osgi plugin)

**Comment by gamerson** [comment]
hello @thugcee  can you share an example project on github that demonstrates the problem along with the steps to reproduce?  thanks.

**Comment by bjhargrave** [comment]
> Bndtools Project Nature to standard Java Gradle project (OSGI bundle was built using Gradle.osgi plugin)

The launcher probably assumes the project is part of a Bnd Workspace build (having a cnf folder with configured repositories where the launcher can be located.) Since the project is a vanilla gradle project, it is not a Bnd Workspace build.

**Comment by gamerson** [comment]
i have a workspace without a cnf and when i perform a launch i get the expected error dialog saying why it couldn't launch.  I haven't seen the launcherStatus failure stacktrace the OP mentioned.

**Comment by thugcee** [comment]
I tried to isolate the problem and here are some facts I've gathered:
- problem is independent from my gradle project (I can close the project and problem still exists);
- creating fresh Bnd Workspace (menu File/New/Bnd OSGi Workspace) didn't solve the problem;
- total cleaning of .metadata directory solved the problem of NullPointerException, but when I launched container (Felix + standard bundles from template's _Central_) I've got an empty console;
- so I created a new bndrun using template Felix4+ and this one started correctly giving my gogo shell;
- when I tried to copy step by step the content of new bndrun to the old one, launching of both configurations stopped working at some point with an error dialog and message: 
```Launch configuration nsg_test.bndrun references non-existing project cnf.```, 
where ```cnf``` is a valid eclipse project containing cnf of bnd workspace. 

I just want to use bndtools to start containers from inside of Eclipse (since building bundles works fine using gradle/osgi plugin).

This is too strange for me :)
Let's close the case and leave it as one more mystery of computers.
